platforms/nuttx/CMakeLists.txt

```diff
@@ -137,11 +137,19 @@ CYGPATH(NUTTX_CONFIG_DIR NUTTX_CONFIG_DIR_CYG)
 if((DEFINED ENV{SIGNING_TOOL}) AND (NOT NUTTX_DIR MATCHES "external"))
 	set(PX4_BINARY_OUTPUT ${PX4_BINARY_DIR}/${PX4_CONFIG}_unsigned.bin)
 
-	add_custom_command(OUTPUT ${PX4_BINARY_DIR_REL}/${PX4_CONFIG}.bin
-		COMMAND $ENV{SIGNING_TOOL} $ENV{SIGNING_ARGS} ${PX4_BINARY_OUTPUT} ${PX4_BINARY_DIR}/${PX4_CONFIG}.bin
-		DEPENDS ${PX4_BINARY_OUTPUT}
-		WORKING_DIRECTORY ${PX4_SOURCE_DIR}
-	)
+	if (CONFIG_OPENSBI AND "${PX4_BOARD_LABEL}" STREQUAL "bootloader")
```

Comment:
Modifying this now broke the pixhawk / saluki builds. Need to do this another way. I just quickly hacked this together to see if this apporach works, and it does.